index.html

```diff
@@ -24,5 +24,146 @@ <h1 class="headline">Charlie Giorgania</h1>
+                <ul class="list">
+                    <li class="project-link">HTML/CSS</li>
+                    <li class="project-link">Ruby on Rails</li>
+                    <li class="project-link">JavaScript</li>
+                </ul>
+            </div>
+            <button class="project-button">See this project<span><img src="./images/ic_arrow_right.png" alt=""></span></button>
+        </article>
+
+        <article class="project-container">
+            
+            <img class="img" src="./images/Rectangle2.png" alt="Project name goes here">
+            <h4 class="project-tittle">Project name goes here</h4>
+            
+            <div class="project-skills container">
+                <ul class="list">
+                    <li class="project-link">HTML/CSS</li>
+                    <li class="project-link">Ruby on Rails</li>
+                    <li class="project-link">JavaScript</li>
+                </ul>
+            </div>
+            <button class="project-button">See this project<span><img src="./images/ic_arrow_right.png" alt=""></span></button>
+        </article>
+
+        <article class="project-container">
+
+            <img class="img" src="./images/Rectangle3.png" alt="Project name goes here">
+            <h4 class="project-tittle">Project name goes here</h4>
+            
+            <div class="project-skills container">
+                <ul class="list">
+                    <li class="project-link">HTML/CSS</li>
+                    <li class="project-link">Ruby on Rails</li>
+                    <li class="project-link">JavaScript</li>
+                </ul>
+            </div>
+            <button class="project-button">See this project<span><img src="./images/ic_arrow_right.png" alt=""></span></button>
+        </article>
+
+        <article class="project-container">
+
+            <img class="img" src="./images/Rectangle4.png" alt="Project name goes here">
+            <h4 class="project-tittle">Project name goes here</h4>
+            
+            <div class="project-skills container">
+                <ul class="list">
+                    <li class="project-link">HTML/CSS</li>
+                    <li class="project-link">Ruby on Rails</li>
+                    <li class="project-link">JavaScript</li>
+                </ul>
+            </div>
+            <button class="project-button">See this project<span><img src="./images/ic_arrow_right.png" alt=""></span></button>
+        </article>
+
+        <article  class="project-container">
+
+            <img class="img" src="./images/Rectangle5.png" alt="Project name goes here">
+            <h4 class="project-tittle">Project name goes here</h4>
+            
+            <div class="project-skills containe">
+                <button class="project-link">HTML/CSS</button>
+                <button class="project-link">Ruby on Rails</button>
+                <button class="project-link">JavaScript</button>
```

Comment:
kindly, I suggest following the advice of the previous reviewer and use UL/LI lists to organize languages